testng-core-api/src/main/java/org/testng/internal/RuntimeBehavior.java

```diff
@@ -127,4 +127,14 @@ public static boolean enforceThreadAffinity() {
   public static int getDefaultVerboseLevel() {
     return Integer.getInteger(TESTNG_DEFAULT_VERBOSE, 1);
   }
+
+  /**
+   * If enabled we will do a extra sort which will improve the ordering of the methods, but might
+   * take a bit longer time.
+   *
+   * @return true if enabled
+   */
+  public static boolean isSecondSortEnabled() {
```

Comment:
Can we rename this JVM argument to something else for better readability ? Maybe `testng.hierarchical.sort.enabled` ? On a side note I think it would be better if this comes in as a configuration instead of a JVM argument (i.e., be available to turn on or off via `CommandLineArgs`)

Reply:
Agree